Add Mini Apps series (#3938)
GoogleChrome/web.dev: src/site/content/en/mini-apps/mini-app-standardization/index.md

---
layout: post
title: Mini app standardization
authors:
  - thomassteiner
date: 2021-01-25
# updated: 2021-01-25
description: |
  This chapter introduces the standardization effort that has been started for mini apps.
tags:
  - mini-apps
---

## Mini app popularity

Mini apps have seen tremendous growth. WeChat mini apps as of June 2020 have reached
[830 million active users](https://www.questmobile.com.cn/research/report-new/122), Alipay mini apps
[401 million active users](https://kr-asia.com/the-mau-of-wechat-alipay-and-baidus-mini-programs-now-add-up-to-more-than-1-billion)
as of April 2019, and Baidu mini apps in the same month
[115 million active users](https://kr-asia.com/the-mau-of-wechat-alipay-and-baidus-mini-programs-now-add-up-to-more-than-1-billion).
Effectively companies have traded building apps for the two operating systems iOS and Android and

> additionally the Web for building apps for three or more super apps platforms. The differences

Author: This was introduced before. Let me place a link again.

> additionally the Web for building apps for three or more super apps platforms. The differences

Author: ```suggestion
additionally the Web for building apps for three or more
[super apps platforms](/what-are-super-apps/#what-are-super-apps).
The differences
```

---
layout: post
title: Mini app standardization
authors:
  - thomassteiner
date:  2021-02-05
# updated: 2021-01-25
description: |
  This chapter introduces the standardization effort that has been started for mini apps.
tags:
  - mini-apps
---

> 

Reviewer: This one could also use an intro: what you've covered so far and what you'll cover now

Author: I have solved this in https://github.com/GoogleChrome/web.dev/pull/3938/commits/9b1f6c5d6f25c7117132deef98f3c44472eca689 by adding an aside to each chapter apart the first that reads:

```
This post is part of an article collection where each article builds upon previous articles. If you just landed here, you may want to start reading from the beginning.
```

- Manifest: [spec](https://w3c.github.io/miniapp/specs/manifest/),
  [explainer](https://github.com/w3c/miniapp/blob/gh-pages/specs/manifest/docs/explainer.md)
- Packaging: [spec](https://w3c.github.io/miniapp/specs/packaging/),
  [explainer](https://github.com/w3c/miniapp/blob/gh-pages/specs/packaging/docs/explainer.md)
- An exploration of [widget requirements](https://w3c.github.io/miniapp/specs/widget-req/)

W3C member and group participant Fuqiao Xue (W3C) has further published a
[Comparison of MiniApps and web apps](https://xfq.github.io/miniapp-comparison/) on his own behalf,
that is, not as an official group publication, but nonetheless worth the read.

<figure class="w-figure">
  <img src="miniapps-whitepaper.png" alt="The header of the MiniApp Standardization White Paper in a browser window.">
  <figcaption class="w-figure">
    The MiniApp Standardization White Paper.
  </figcaption>
</figure>

## Formal launch of the W3C MiniApps Working Group

On January 19, 2021, the [MiniApps Working Group](https://www.w3.org/2021/miniapps/) was [formally launched](https://www.w3.org/blog/2021/01/w3c-launches-the-miniapps-working-group/) in the W3C.

> The group uses the spelling "MiniApps" to distinguish the standardization effort from the technology.

Reviewer: ```suggestion
The group uses the spelling and capitalization "MiniApps" to distinguish the standardization effort from the technology.
```

> You can read the group's [charter](https://www.w3.org/2021/01/miniapps-wg-charter.html) to get a feeling of the planned work.

Reviewer: ```suggestion
You can read the group's [charter](https://www.w3.org/2021/01/miniapps-wg-charter.html) to get a feel for the planned work.
```
This is the more common phrasing
